src/tests/suite/dfs_unit_test.c

```diff
@@ -2212,15 +2227,22 @@ dfs_setup(void **state)
 	arg = *state;
 
 	if (arg->myrank == 0) {
-		bool	use_dtx = false;
+		bool		use_dtx = false;
+		dfs_attr_t	attr = {};
 
 		d_getenv_bool("DFS_USE_DTX", &use_dtx);
 		if (use_dtx)
 			print_message("Running DFS Serial tests with DTX enabled\n");
 		else
 			print_message("Running DFS Serial tests with DTX disabled\n");
 
-		rc = dfs_cont_create(arg->pool.poh, &co_uuid, NULL, &co_hdl, &dfs_mt);
+		attr.da_props = daos_prop_alloc(1);
+		assert_non_null(attr.da_props);
+		attr.da_props->dpp_entries[0].dpe_type = DAOS_PROP_CO_REDUN_LVL;
+		attr.da_props->dpp_entries[0].dpe_val = DAOS_PROP_CO_REDUN_RANK;
```

Comment:
that is not the default? i would rather we use the default with this test TBH as that what most users would use.

Reply:
default is node, not rank. Actually this is mostly for running a test with many engines over a single node. Anyway, this is probably out of scope of this ticket.